client/ayon_houdini/plugins/create/create_review.py

```diff
@@ -29,8 +30,15 @@ def apply_settings(self, project_settings):
             self.review_color_space = color_settings.get("review_color_space")
 
     def create(self, product_name, instance_data, pre_create_data):
-
-        instance_data.update({"node_type": "opengl"})
+        
+        self.node_type = pre_create_data.get("node_type")
+        hou_version = hou.applicationVersionString()
+        if self.node_type == "flipbook" and not hou_version.startswith("20.5"):
+            self.log.debug("The Flipbook node type isn't supported in Houdini versions below 20.5."
+                           " Switching to the OpenGL node type instead.")
+            self.node_type = "opengl"
```

Comment:
Let's remove this here - and just keep the `flipbook` entry out of the `Enum`? if `houdini < 20.5`

Reply:
Also, we should not be updating the `self.node_type` value - it makes no sense to change the class attribute from here.

Reply:
https://github.com/ynput/ayon-houdini/pull/167/commits/9bb792ead39c988edc78f19342dcfeba8c62fa94